server/src/main/java/io/crate/planner/operators/InsertFromValues.java

```diff
@@ -488,20 +486,6 @@ public List<CompletableFuture<Long>> executeBulk(DependencyCarrier dependencies,
         );
     }
 
-    private static void checkPrimaryKeyValuesNotNull(ArrayList<Input<?>> primaryKeyInputs) {
-        for (var primaryKey : primaryKeyInputs) {
-            if (primaryKey.value() == null) {
-                throw new IllegalArgumentException("Primary key value must not be NULL");
-            }
-        }
```

Comment:
Looks like RowShardResolver doesn't do NULL check for multiple PK-s, i.e it's another bug in insert-from-subquery. 
Will push fix asap. 

Similar to CLUSTERED BY, it will be reused by InsertFromValues as well.

Reply:
No, in case of multiple PK-s, `Id.encode` is used and does null check. it even des null check for CLUSTERED BY which this PR is about but only in case of multiple primary keys and clustered by being